covid_moonshot/analysis/free_energy.py

```diff
@@ -1,52 +1,57 @@
 import functools
-from typing import List, Optional
+from typing import List, Optional, Tuple
 import numpy as np
 from pymbar import BAR
 from pymbar.mbar import MBAR
-from covid_moonshot.core import PhaseAnalysis, Work
+from covid_moonshot.core import FreeEnergy, GenAnalysis, PhaseAnalysis, Work
 
 
-def mask_outliers(a: np.ndarray, max_value: float, n_devs: float) -> np.ndarray:
+def mask_outliers(a: np.ndarray, max_value: float, max_n_devs: float) -> np.ndarray:
     """Returns a boolean array masking values that are more than
     `n_devs` standard deviations from the mean or larger in magnitude
     than `max_value`.
 
     Parameters
     ----------
-    a : array
+    a : array_like
     max_value : float
-        Values larger (in magnitude) than this will be discarded
-    n_devs : float
-        Values farther than this number of standard deviations from
-        the mean will be discarded
+        Remove values with magnitudes greater than this
+    max_n_devs : float
+        Remove values farther than this number of standard deviations
+        from the mean
 
     Returns
     -------
-    array of bool
+    ndarray of bool
```

Comment:
Can we mark the expected dimensions (shapes) of ndarrays in docstrings too, where convenient?

Reply:
Added in 793b18e.